personalize.admin.inc

```diff
@@ -205,7 +205,7 @@ function personalize_admin_build_visitor_context_select($settings, $exclude_disa
   }
   $element = array(
     '#type' => 'select',
-    '#title' => t('Visitor contexts to use for automatic targeting'),
+    '#title' => t('Visitor contexts'),
```

Comment:
Oh right - forgot we still want to use the disabled context config to prevent certain contexts from showing up for fixed targeting.